src/main/java/sirius/db/es/Elastic.java

```diff
@@ -84,10 +86,15 @@
     @ConfigValue("elasticsearch.hosts")
     private String hosts;
 
+    @ConfigValue("elasticsearch.logQueryThreshold")
```

Comment:
typo log <=> long

Reply:
no this is the threshold to determine if the query should be logged^^

Reply:
Then the typo is in the field name